From: Markus Wanner <markus@bluegap.ch>
Date: Mon, 10 Feb 2014 13:37:04 +0100
Subject: Add default directories used by Debian

If not given, fgrun asks the user to provide several directories. We
ease this process by defaulting to the install locations used by Debian.
---
 src/main.cxx | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/main.cxx b/src/main.cxx
index c9aee92..cf51f07 100644
--- a/src/main.cxx
+++ b/src/main.cxx
@@ -37,10 +37,10 @@
 #include "wizard.h"
 #include "i18n.h"
 
-std::string def_fg_exe = "";
-std::string def_fg_root = "";
-std::string def_fg_scenery = "";
-std::string def_ts_exe = "";
+std::string def_fg_exe = "/usr/games/fgfs";
+std::string def_fg_root = "/usr/share/games/flightgear";
+std::string def_fg_scenery = "/usr/share/games/flightgear/Scenery";
+std::string def_ts_exe = "/usr/games/terrasync";
 std::string version = "";
 std::string def_ts_dir = "";
 std::string def_acft_dir = "";
